docs/src/user-guide/quick-start/clp-text.md

```diff
@@ -0,0 +1,153 @@
+# clp-text quick-start
+
+This page will walk you through how to start CLP and use it to compress and search unstructured
+text logs.
+
+:::{caution}
+If you're using a `clp-text` release, you should only compress unstructured text logs. `clp-text`
+is able to compress and search JSON logs as if it was unstructured text, but `clp-text` cannot
+query individual fields. This limitation will be addressed in a future version of CLP.
+:::
+
+---
+
+## Starting CLP
+
+To start CLP, run:
+
+```bash
+sbin/start-clp.sh
+```
+
+:::{note}
+If CLP fails to start (e.g., due to a port conflict), try adjusting the settings in
+`etc/clp-config.yml` and then run the start command again.
+:::
+
+---
+
+## Compressing unstructured text logs
+
+To compress some unstructured text logs, run:
+
+```bash
+sbin/compress.sh <path1> [<path2> ...]
+```
+
+`<path...>` are paths to unstructured text log files or directories containing such files.
+
+The compression script will output the compression ratio of each dataset you compress, or you can
+use the UI to view overall statistics.
+
+Compressed logs will be stored in the directory specified by the `archive_output.storage.directory`
+config option in `etc/clp-config.yml` (`archive_output.storage.directory` defaults to
+`var/data/archives`).
+
+### Sample logs
+
+For some sample logs, check out the [open-source datasets](../resources-datasets).
+
+---
+
+## Searching unstructured text logs
+
+You can search your compressed logs from CLP's [UI](#searching-from-the-ui) or the
+[command line](#searching-from-the-command-line).
+
+Unstructured text log queries are written as plain text. You can use a couple of special
+characters to make these queries more versatile. `*` can be used as a placeholder for an unknown
+number of characters, and `?` can be used for a single character. For example, consider the logs in
+[Figure 1](#figure-1) and the query in [Figure 2](#figure-2).
+
+(figure-1)=
+:::{card}
+
+```text
+1   abc
+2   axbc
+3   abxc
+4   axxbxc
+5   a b c
+```
+
++++
+**Figure 1**: A set of unstructured text log events.
+:::
+
+(figure-2)=
+:::{card}
+
+```bash
+"a*b?c"
+```
+
++++
+**Figure 2**: An example query.
+:::
```

Comment:
Can we use the logs we have in the flavor selection page? And perhaps a more realistic query.